x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTests.java

```diff
@@ -2644,7 +2644,6 @@ private void assertEmptyEsRelation(LogicalPlan plan) {
         assertThat(plan, instanceOf(EsRelation.class));
         EsRelation esRelation = (EsRelation) plan;
         assertThat(esRelation.output(), equalTo(NO_FIELDS));
-        assertTrue(esRelation.index().mapping().isEmpty());
```

Comment:
Not sure if it is possible to replace this assertion in `assertEmptyEsRelation` with any alternative?